src/domUpdates.js

```diff
@@ -104,33 +104,17 @@ class DomUpdates {
     })
   }
 
-  updateSearchByRecipeName(inputByUser) {
-    const recipeCards = document.querySelectorAll('.recipe-name')
-    const lowerInput = inputByUser.toLowerCase()
```

Comment:
I noticed this change and I'd like to know more about this, I noticed that now when the user searches for a recipe by ing name, the result it is case sensitive.  What am I missing here? 
why was the change needed?